src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -498,6 +505,9 @@ def load_arguments(self, _):
                 target="--uptime-sla", redirect="--tier", hide=True
             ),
         )
+        c.argument(
+            "sku", is_preview=True, arg_type=get_enum_type(sku_names), validator=validate_sku_name
```

Comment:
As you've declared this is an enum type, arg parse would help deal with the validation and you could safely omit the validator as it's doing the same thing.

Reply:
I just tried to align with the writing format for "tier". Because I want to raise an error to alert user that --sku can only be base or automatic just as the same thing tier does, --tier can only be free, standard, or premium. Do you think it must remove it or it's okay to keep there for the validator_sku_name.